iap/README.md

```diff
@@ -76,6 +76,26 @@ service account private key can impersonate that account!
    ```
 2. Copy `validate_jwt.py` into your application.
 
+## Using generate_self_signed_jwt
+
+### Self-signed JWT with IAM Credentials API
+
+1. Install the libraries listed in `requirements.txt`, e.g. by running:
```

Comment:
Can you specify which `requirements.txt` file? This repository is huge, and can quite be confusing for first time users. Consider linking directly to the file you're referring to.